src/App.js

```diff
@@ -82,7 +82,7 @@ const App = () => {
       body: JSON.stringify(newExpense),
     };
 
-    // Reset alerts in case user did not dismiss them
+    // Reset alerts in case user did not dismiss them. Still necessary?
     setExpenseAdded(undefined);
     setExpenseDeleted(undefined);
```

Comment:
Since the alerts now disappear automatically after 5 seconds, does it makes sense to still reset these state variables here?  Also, if I understand correctly, only the green positive alerts disappear automatically — the catch block alerts (the red ones ala "There expense could not be added") still need to be closed manually. Do we maybe want to have them all disappear automatically?

Reply:
from a product perspective: consistency is king, so please let us have all messages disappear automatically

Reply:
from a development perspective: i really dislike this `setExpenseAdded(undefined)` call. as mentioned in a prior discussion, i find it not particularly descriptive. i would be happy to see it go away :)